include/boost/beast/experimental/core/impl/url_parser.ipp

```diff
@@ -0,0 +1,352 @@
+//
+// Copyright (c) 2018 jackarain (jack dot wgm at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/beast
+//
+
+#include <boost/beast/experimental/core/url_parser.hpp>
+
+namespace boost {
+namespace beast {
+
+//--------------------------------------------------------------------------
+// scheme:[//[user[:password]@]host[:port]][/path][?query][#fragment]
+
+using boost::beast::string_view;
```

Comment:
Hmm.. probably better not to have a `using` statement at namespace scope. `string_view` is already a symbol in Beast you should be able to remove this `using` statement and still have everything compile.